<launch>
	<arg name="stream" default="False" />
	<arc name="polar" default="False" />
	<node pkg="sonar" name="sonar_pub" type="sonar_publisher.py">
		<param name="stream" value="$(arg stream)" />
		<param name="polar" value="$(arg polar)" />
	</node>
</launch>
